packages/bot-web-ui/src/components/dashboard/tutorial-tab/config/index.ts

```diff
@@ -463,7 +448,33 @@ export const faq_content: TFaqContent[] = [
                 ),
             },
         ],
-        tab_id: 2,
         search_id: 'faq-17',
     },
 ];
+
+export const quick_strategy_content: TQuickStrategyGuides[] = [
```

Comment:
@farabi-deriv I think it's the short description that we are showing on the tutorial page. In that case, will it be okay to name this variable `quick_strategy_content` to `qs_tutorial_short_descriptions` or something similar?

Reply:
@shafin-deriv its not really short descriptions, its the thumbnail of the strategy contents that lead to the long descriptions. I've kept the naming consistent with the rest in tutorial tab. As we have `guide_content` and `faq_content`, I've added the `quick_strategy_content` in this config.